UPGRADE.md

```diff
@@ -59,6 +59,9 @@ There is a list of all the repositories maintained by monorepo, changes in log b
     - please read upgrade instructions in [separate article](docs/upgrade/services_removal.md)
 
 ### [shopsys/project-base]
+- [#633 Google Cloud deploy using Terraform, Kustomize and Kubernetes](https://github.com/shopsys/shopsys/pull/633)
```

Comment:
This is all upgrade instructions? I see some changes in kubernetes configuration for CI. Should there be also instructions for users that are using Kuberenetes for CI?